src/app/page/Recipient/Dashboard.tsx

```diff
@@ -42,27 +43,49 @@ const completedStatus = [MissionStatus.delivered, MissionStatus.succeeded];
 export default function () {
   const classes = useStyles();
   const location = useLocation();
-  // @ts-ignore no types for state
-  const auth = useSelector((state) => state.firebase.auth);
   const [missions, setMissions] = useState<{
     submitted: MissionInterface[];
     completed: MissionInterface[];
   }>({ submitted: [], completed: [] });
 
+  // @ts-ignore no types for state
+  const auth = useSelector((state) => state.firebase.auth);
+  const org = useOrganization();
+
+  useFirestoreConnect(() => {
```

Comment:
swapping out the `getAllRequestedMissions` call for this so we can update on feedback changes